### ISSUE TITLE
How to use the REST API from an external client

### ISSUE THREAD

**Issue by tofferlicious** [nextcloud/notes#161]
I am trying to access the API via Browser and with different REST clients as described here:
https://github.com/nextcloud/notes/wiki/API-0.2

Hence calling the interface with this url scheme including the user and password:
https://user:password@yournextcloud.com/index.php/apps/notes/api/v0.2/

But in the Browser I get:
`{"message":"CORS requires basic auth"}`

When adding basic authentication with a rest client (like Postman), it also does not work.

Basically what I want to do, is to update notes on my Nextcloud server from a python script running on a different machine.

**Comment by tofferlicious** [comment]
Ok, got it working by adding the Header
`"OCS-APIRequest: true"` 
and using Basic Authentication.
Documentation could be updated maybe. I was using NC12 by the way :)

**Comment by Aliyss** [comment]
> Ok, got it working by adding the Header
> `"OCS-APIRequest: true"`
> and using Basic Authentication.
> Documentation could be updated maybe. I was using NC12 by the way :)

If I may ask... How did you you connect without getting a cors error?

**Comment by GerardPolloRebozado** [comment]
> Ok, got it working by adding the Header `"OCS-APIRequest: true"` and using Basic Authentication. Documentation could be updated maybe. I was using NC12 by the way :)

I ahve TOTP enabled and using basic gives me Password login forbiden and using the token gives me the cors error I tried adding OCS-APIRequest: true gives error too.
